cmd/config/config.go

```diff
@@ -0,0 +1,54 @@
+package config
+
+import (
+	"log"
+	"os"
+	"path/filepath"
+
+	"github.com/mitchellh/go-homedir"
+	"github.com/spf13/cobra"
+)
+
+const (
+	configFileName = "config"
+	configDirName  = ".sdlocal"
+)
+
+var filePath = func(isLocalOpt bool) (string, error) {
+	if isLocalOpt {
+		pwd, err := os.Getwd()
+		if err != nil {
+			return "", err
+		}
+		return filepath.Join(pwd, configDirName, configFileName), nil
+	}
+	home, err := homedir.Dir()
+	if err != nil {
+		return "", err
+	}
+	return filepath.Join(home, configDirName, configFileName), nil
+}
+
+// NewConfigCmd return config command.
+func NewConfigCmd() *cobra.Command {
+	configCmd := &cobra.Command{
+		Use:   "config",
+		Short: "Can settings related to sd-local.",
```

Comment:
A description does not start with verbs looks weird for me. (refs. kubectl, docker)

Reply:
thx.
I fixed it.